### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by psjnek**

### CODE REVIEW

File: Languages/german.lang [hunk 1]
```diff
@@ -285,7 +285,7 @@ msgid "Block IOS Reload"
 msgstr "Blocke IOS-Neuladen"
 
 msgid "Block Loader Layout Button"
-msgstr ""
+msgstr "Blocke Loader Layout Button"
```
Comment (eku) [inline Languages/german.lang]: `Loader Layout Button` is not Deutsch.

File: Languages/german.lang [hunk 2]
```diff
@@ -321,7 +321,7 @@ msgid "Block Updates"
 msgstr "Blocke Updates"
 
 msgid "Boot Content"
-msgstr "Boote Inhalt"
+msgstr "Boot-Inhalt"
```
Comment (eku) [inline Languages/german.lang]: `Boot` is an action.

File: Languages/german.lang (hunk 1 above)
Comment (psjnek) [reply Languages/german.lang]: I know, I could not find a better word for it...

Comment (eku) [reply Languages/german.lang]: Well, @wiidev might explain what `Block Loader Layout Button"` actualy does /mean.

Comment (psjnek) [reply Languages/german.lang]: The translations below pause use a similar wording. That’s why I translated it that way.

File: Languages/german.lang (hunk 2 above)
Comment (psjnek) [reply Languages/german.lang]: I changed it to your suggestion a few weeks ago already but it does not show up in "Files changed".

File: Languages/german.lang [hunk 3]
```diff
@@ -410,7 +410,7 @@ msgid "Cheatfile is blank"
 msgstr "Cheatdatei ist leer"
 
 msgid "Clear"
-msgstr "Freigeben"
+msgstr "Löschen"
```
Comment (eku) [inline Languages/german.lang:413]: Please check again the context where `Clear` ist used. IMHO the translation is correct.

**Review by eku — CHANGES_REQUESTED** [review]

File: Languages/german.lang (hunk 3 above)
Comment (psjnek) [reply Languages/german.lang:413]: "clear" is used in the GUI settings to clear the input. I can send a screenshot if you need one.

Comment (psjnek) [reply Languages/german.lang:413]: > Please check again the context where `Clear` is used. IMHO the translation is correct.

Here is the screenshot:

Comment (psjnek) [reply Languages/german.lang:413]: I think "Freigeben" does not make any sense in this context.

**Comment by WebSnke** [comment]
@wiidev 
I think this PR is ready to be merged.

File: Languages/german.lang (hunk 3 above)
Comment (WebSnke) [reply Languages/german.lang:413]: @eku 
Could you maybe commit my suggestion?

**Comment by psjnek** [comment]
@wiidev 
Could you maybe merge my pull request?

**Pull request closed without merging**